templates/vue/src/utils/Helpers.js

```diff
@@ -188,6 +188,14 @@ export default class Helpers {
       return wp.canEditTapestry()
     }
 
+    // Check 2: User submitted node, can't edit, but can move
+    if (node.author && wp.isCurrentUser(node.author.id) && action == "move") {
+      if (node.reviewStatus !== nodeStatus.ACCEPT) {
+        return true
+      }
+    }
```

Comment:
This should be moved down below into the `if (node.status === nodeStatus.DRAFT)` conditional and the comment block for that conditional should also be updated.